src/chains/ethereum/ethereum/tests/api/evm/snapshot.test.ts

```diff
@@ -82,7 +82,7 @@ describe("api", () => {
 
         // Assert the starting balance is where we think it is, including tx costs.
         assert(
-          balance > 98900000000000000000 && balance < 99000000000000000000
+          balance > 990000000000000000000n && balance < 999000000000000000000n
```

Comment:
Why do we test against a range here? Is it not totally deterministic?

Reply:
This is one of the tests that was ported from the old tests, not sure why it was written this way.

Reply:
Should I go ahead and make a specific assertion here?